src/Cli/dotnet/commands/dotnet-workload/update/WorkloadUpdateCommand.cs

```diff
@@ -57,7 +57,7 @@ internal class WorkloadUpdateCommand : InstallingWorkloadCommand
                                 elevationRequired: !_printDownloadLinkOnly && !_printRollbackDefinitionOnly && string.IsNullOrWhiteSpace(_downloadToCacheOption));
 
             _workloadManifestUpdater = _workloadManifestUpdaterFromConstructor ?? new WorkloadManifestUpdater(Reporter, workloadResolver ?? _workloadResolver, PackageDownloader, _userProfileDir, TempDirectoryPath,
-                _workloadInstaller.GetWorkloadInstallationRecordRepository(), _workloadInstaller, _packageSourceLocation);
+                _workloadInstaller.GetWorkloadInstallationRecordRepository(), _workloadInstaller, _packageSourceLocation, null, true, _sdkFeatureBand);
```

Comment:
Nit: you could use a named parameter to avoid adding the other parameters where you're just passing the default values: 

```suggestion
                _workloadInstaller.GetWorkloadInstallationRecordRepository(), _workloadInstaller, _packageSourceLocation, sdkFeatureBand: _sdkFeatureBand);
```